composer.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "burdamagazinorg/access_unpublished",
```

Comment:
I guess we have to use drupal/access_unpublished, because we will push it to d.o. in the next days